iceoryx_posh/test/moduletests/test_posh_runtime_node.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Comment:
Here we have to coordinate better since @FerdinandSpitzschnueffler is currently working on the nodes as you can see here: #349 

In the future you could inform us on which tests you would like to work so that not multiple developer working on the same problem at the same time.

Reply:
Hi, Actually I am not responsable on which folder we are working so far. Maybe, you should align with @nihalchari  :)

Reply:
No problem here. The Node class will perhaps be extended in the future but this will probably not influence the existing methods.